src/xml_db/parse/entry.rs

```diff
@@ -0,0 +1,481 @@
+                        if let Some(value) = field.value {
+                            out.fields.insert(field.key, value);
+                        }
+                    }
+                    "CustomData" => {
+                        let value = CustomData::from_xml(iterator, inner_cipher)?;
+                        // TODO
+                    }
+                    "Binary" => {
+                        let field = BinaryField::from_xml(iterator, inner_cipher)?;
+                        // TODO reference into a binary field from the Meta. Might only appear in
+                        // kdbx3
+                    }
+                    "AutoType" => {
+                        out.autotype = Some(AutoType::from_xml(iterator, inner_cipher)?);
+                    }
+                    "Times" => {
+                        out.times = Times::from_xml(iterator, inner_cipher)?;
+                    }
+                    "IconID" => {
+                        let icon_id = SimpleTag::<usize>::from_xml(iterator, inner_cipher)?;
+                        // TODO
+                        // out.icon_id = icon_id;
+                    }
+                    "CustomIconUUID" => {
+                        let icon_id = SimpleTag::<String>::from_xml(iterator, inner_cipher)?.value;
+                        // TODO
+                    }
+                    "ForegroundColor" => {
+                        let color = SimpleTag::<Option<String>>::from_xml(iterator, inner_cipher)?;
+                        // TODO
+                        // out.foregrpund_color = color;
+                    }
+                    "BackgroundColor" => {
+                        let color = SimpleTag::<Option<String>>::from_xml(iterator, inner_cipher)?;
+                        // TODO
+                        // out.background_color = color;
+                    }
+                    "OverrideURL" => {
+                        let url = SimpleTag::<Option<String>>::from_xml(iterator, inner_cipher)?;
+                        // TODO
+                        // out.override_url = color;
+                    }
+                    "QualityCheck" => {
+                        let qc = SimpleTag::<bool>::from_xml(iterator, inner_cipher)?;
+                        // TODO
+                        // out.quality_check = qc;
+                    }
+                    "History" => {
+                        let history = History::from_xml(iterator, inner_cipher)?;
+                        // TODO
+                        // out.history = history;
+                    }
+                    _ => {
+                        return Err(XmlParseError::BadEvent {
+                            expected: "valid Entry child",
+                            event: event.clone(),
+                        })
+                    }
+                },
+                SimpleXmlEvent::End(name) if name == "Entry" => break,
+                _ => {
```

Comment:
Added the `IgnoreSubfield` parser that can be used to ignore unknown fields.

Reply:
@sseemayer is it already used? Would be nice to add a unit test to make sure we don't throw an error in case of unknown field. We could also add an option in the future to enable `strict` parsing, otherwise we always risk losing information when saving if the field is not supported.